master/internal/scheduler/kubernetes_resource_provider.go

```diff
@@ -378,7 +378,7 @@ func (k *kubernetesResourceProvider) taskTerminated(task *Task, aborted bool) {
 	}
 }
 
-func (k *kubernetesResourceProvider) receiveTaskStopped(ctx *actor.Context, msg taskStopped) {
+func (k *kubernetesResourceProvider) receiveTaskStopped(ctx *actor.Context, msg taskActorStopped) {
```

Comment:
non-blocking: i would probably update the commit name since it only mentions the default RP

Reply:
sure. I'll update that when I merge it.